src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -511,6 +511,11 @@ event::ptr primitive_inst::realloc_if_needed() {
         }
     }
 
+    if (_node->is_type<crop>() && can_be_optimized()) {
+        GPU_DEBUG_PROFILED_STAGE_MEMALLOC_INFO("can_be_optimized");
+        return ev;
+    }
```

Comment:
I think you should add (crop) to line 612 (is_runtime_optimized) instead of this.

Reply:
Added crop to condition to check `is_runtime_skippable`